src/components/webview/MainProvider.ts

```diff
@@ -29,10 +30,38 @@ import {
 	CodemodNodeHashDigest,
 	selectCodemodArguments,
 } from '../../selectors/selectCodemodTree';
-import { isNeitherNullNorUndefined } from '../../utilities';
+import { buildTypeCodec, isNeitherNullNorUndefined } from '../../utilities';
 
 const X_INTUITA_ACCESS_TOKEN = 'X-Intuita-Access-Token'.toLocaleLowerCase();
 
+export const validateAccessToken = async (
+	accessToken: string,
+): Promise<boolean> => {
+	const result = await axios.post(
+		'https://telemetry.intuita.io/validateAccessToken',
+		{},
+		{
+			headers: {
+				[X_INTUITA_ACCESS_TOKEN]: accessToken,
+			},
+			timeout: 5000,
+		},
+	);
+	const { data } = result;
+
+	const validation = buildTypeCodec({
```

Comment:
You don't need the body validation at all right now, right?

Reply:
addressed